src/variables.less

```diff
@@ -0,0 +1,4 @@
+@headerHeight: 50px;
```

Comment:
IMHO we should use css custom properties instead of less variables as they don't need import statements and can be modified at runtime.